docs/mediapipe.md

```diff
@@ -191,34 +177,49 @@ Nodes in the MediaPipe graphs can reference both to the models configured in mod
 
 Subconfig file may only contain *model_config_list* section  - in the same format as in [models config file](starting_server.md).
 
-## Using MediaPipe <a name="using-mediapipe"></a>
 
-MediaPipe graphs can use the same KServe Inference API as the models. There are exactly the same calls for running
-the predictions. The request format must match the pipeline definition inputs.
+## Deployment testing <a name="testing"></a>
+### Debug logs
+The simples method to validate the graph execution is to set the Model Server `log_level` do `TRACE`.
+`docker run --rm -it -v $(pwd):/config openvino/model_server:latest --config_path /config/config.json --log_level TRACE`
 
-Graphs can be queried for their state using the calls [GetModelStatus](model_server_grpc_api_kfs.md)
-and [REST Model Status](model_server_rest_api_kfs.md)
+It will report in a verbose way all the operations in the mediapipe framework from the graph initialization and execution.
+The model server logs could confirm the graph correct format and loading all the required models.
+Note that graph definition loading is not confirming if all the calculators are compiled into the model server. That can be tested after sending the request to the KServe endpoint.
+During the requests processing, the logs will include info about calculators registration and processing the nodes.
```

Comment:
```suggestion
During the requests processing, the logs will include infoabout calculators registration and processing the nodes.
```
There is no info that calculator is registered - only in case of error we expect message from MP to inform us that sth is missing.